src/jit/compiler.cpp

```diff
@@ -49,6 +49,13 @@ AssemblyNamesList2* Compiler::s_pAltJitExcludeAssembliesList            = nullpt
 #endif // ALT_JIT
 
 /*****************************************************************************/
+#ifdef _TARGET_AMD64_
+#include <intrin.h>
+inline bool _our_GetThreadCycles(unsigned __int64 *cycleOut){ *cycleOut = __rdtsc(); return true; }
```

Comment:
To make this work xplat too, it seems you might want to use what we have in gc.cpp - see the get_cycle_count().